Jenkinsfile

```diff
@@ -8,7 +8,7 @@ elifePipeline {
     stage 'Project tests', {
         lock('journal--ci') {
             builderDeployRevision 'journal--ci', commit
-            builderProjectTests 'journal--ci', '/srv/journal', ['/srv/journal/build/phpunit.xml', '/srv/journal/build/behat.xml'], ['smoke', 'project']
+            builderProjectTests 'journal--ci', '/srv/journal', ['/srv/journal/build/phpunit/*.xml', '/srv/journal/build/behat/*.xml'], ['smoke', 'project']
```

Comment:
Goes to `bldr download_file` and doesn't work?

Reply:
Makes sense, libraries have local artifacts and so Jenkins can see all of the files; these are on `journal--ci` instead so there's an additional SSH indirection to copy them back. I'll open the builder box to see if it can be expanded, or if it needs to compromise by copying the whole folder.

Reply:
Surprisingly `bldr download_file:journal--ci,/srv/journal/build/*.xml,.` works

Reply:
However, the step attempts to create local files with `touch` before downloading them, probably due to permissions problem. It would create only one file with a `*` in the name here.

Reply:
Have some changes to integrate, but they touch upon different steps so need to push them one at a time to remove dependencies first on `builderTestArtifact`, then that step can be refactored.

Reply:
Finally integrated at https://alfred.elifesciences.org/job/pull-requests-projects/job/journal/job/PR-889/17/testReport/